Builds/CMake/CMakeFuncs.cmake

```diff
@@ -147,11 +147,7 @@ macro(setup_build_cache)
     set(CMAKE_CONFIGURATION_TYPES
       DebugClassic
       ReleaseClassic)
-    if(${CMAKE_BUILD_TYPE} STREQUAL "Debug")
-      set(CMAKE_BUILD_TYPE DebugClassic)
-    elseif(${CMAKE_BUILD_TYPE} STREQUAL "Release")
-      set(CMAKE_BUILD_TYPE ReleaseClassic)
-    endif()
+    set(CMAKE_BUILD_TYPE "${CMAKE_BUILD_TYPE}Classic")
```

Comment:
This line is a duplicate!
https://github.com/ximinez/rippled/blob/cmake_assert/Builds/CMake/CMakeFuncs.cmake#L115-L117